exchange/exchangetest/floors_enforcement.json

```diff
@@ -151,6 +151,35 @@
             ]
           }
         ]
+      },
+      "ext": {
+        "prebid": {
+          "seatnonbid": [
+           {
+            "nonbid": [
+              {
+                "impid": "my-imp-id",
+                "statuscode": 301,
```

Comment:
Can you add another json test for the new 304 status code?

Reply:
yes, done.